cypress/e2e/teampage.cy.js

```diff
@@ -0,0 +1,95 @@
+describe('Chosing Drivers Path', () => {
+  beforeEach('', () => {
+    cy.intercept('https://v1.formula-1.api-sports.io/rankings/drivers?season=2021', {
+      fixture: 'allDrivers.json'
+    })
+      .intercept('https://v1.formula-1.api-sports.io/drivers?name=Lewis%20Hamilton', {
+        fixture: 'lewis.json'
+      })
+      .intercept('https://v1.formula-1.api-sports.io/drivers?name=Max%20Verstappen', {
+        fixture: 'max.json'
+      })
+      .intercept('https://v1.formula-1.api-sports.io/rankings/teams?season=2021', {
+        fixture: 'teams2021.json'
+      })
+      .visit('http://localhost:3000/team')
+  });
+
+  it('should ask the user to choose one driver at a time', () => {
+    cy.get('.driver-select').contains('Pick your driver:')
+      .get('select').select('Lewis Hamilton')
+      .get('.submit-driver').click()
+      .get('.driver-select').should('exist');
+  });
+
+  it('should not allow a user to pick the same driver twice', () => {
+    cy.get('select').select('Lewis Hamilton').get('.submit-driver').click()
+      .get('select option:contains(Lewis Hamilton)').should('not.exist');
+  });
+
+  it('should not allow user to submit no driver and ask them to select one', () => {
+    cy.get('.submit-driver').contains('Sign my driver!').click()
+      .get('.submit-driver').contains('Please choose a driver!');
+  });
+
+  it('should only allow user to name the team once both drivers are selected', () => {
+    cy.get('.team-header').should('not.exist')
+      .get('select').select('Lewis Hamilton').get('.submit-driver').click()
+      .get('select').select('Max Verstappen').get('.submit-driver').click()
+      .get('.team-header').should('exist');
+  });
+});
```

Comment:
This is a perfect test! Straightforward, easy to read and exactly tests what it's looking for!